integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/recat/DecisionSpecification.groovy

```diff
@@ -65,6 +65,20 @@ class DecisionSpecification extends GebReportingSpec {
     data.assigned_user_id == ['RECATEGORISER_USER']
   }
 
+  def "Indeterminate offender doesn't display open conditions options (young offender)"() {
+
+    when: 'I go to the decision page'
+    fixture.gotoTasklistRecatForCatJIndeterminate(false)
+    at TasklistRecatPage
+    decisionButton.click()
+
+    then: 'The page is displayed without open condition options'
+    at DecisionPage
+    headerValue*.text() == ['Hillmob, Ant', 'B2345YZ', '01/01/2018', 'J']
+    !categoryJOption.isDisplayed()
```

Comment:
Could you check these assertions work? i found that displayed for radio buttons doesnt work in some cases, i had to use e.g.:
 overriddenCategoryB.@type == 'radio'
 overriddenCategoryJ.@type == null

Reply:
they seem to be reliable for optional radios here